fuel-tx/src/lib.rs

```diff
@@ -98,8 +98,9 @@ pub use transaction::{
     Witness,
 };
 
-#[cfg(feature = "std")]
+#[cfg(feature = "alloc")]
 pub use transaction::Signable;
+#[cfg(feature = "alloc")]
 pub use transaction::UniqueIdentifier;
```

Comment:
Hmm, I think it should work without `alloc` feature

Reply:
Yep, fixed in 6d9229ef